articles/data-share/share-your-data.md

```diff
@@ -67,7 +67,7 @@ Sign in to the [Azure portal](https://portal.azure.com/).
 
 Create an Azure Data Share resource in an Azure resource group.
 
-1. Select the **Create a resource** button (+) in the upper-left corner of the  portal.
+1. Select the menu button in the upper-left corner of the portal, then select **Create a resource** (+).
```

Comment:
This was an additional change I noticed in updating this tutorial, you have click the menu icon first and then select Create a resource. If that's agreed on, i'll go back and update the quickstart article as well to match.